packages/asset-swapper/src/utils/market_operation_utils/types.ts

```diff
@@ -93,6 +93,9 @@ export enum ERC20BridgeSource {
     // Avalanche
     Pangolin = 'Pangolin',
     TraderJoe = 'TraderJoe',
+    // Celo only
+    UbeSwap = 'UbeSwap',
+    UbeSwapMoola = 'UbeSwapMoola',
```

Comment:
I think we should just put these behind the same `UbeSwap` source but select the router based on the tokens being traded.